Python_Scripts/C.py

```diff
@@ -41,15 +45,6 @@
     'SOLVERSTUDY_BIOMODELS_MODEL_FILE', 'biomodels_models.txt')
 INCLUDE_FROEHLICH = os.getenv('SOLVERSTUDY_INCLUDE_FROEHLICH', 'YES')
 
-###############################################################################
-# Data directories
-
-# Simulation data base folder
-DIR_DATA = os.path.join(DIR_BASE, 'Data')
-# Folder for most of the sub-studies
-DIR_DATA_WHOLESTUDY = os.path.join(DIR_DATA, 'WholeStudy')
-# Folder for the tolerances sub-study
-DIR_DATA_TOLERANCES = os.path.join(DIR_DATA, 'TolerancesStudy')
```

Comment:
yes, well... I didn't take the time to look for all the things that can be cleaned up. Thanks for taking care of that!

Reply:
yes, basically you renamed "Data" to "results", which also makes more sense, so I updated that. all paths in plots yet to be updated, but those need extra care anyway.

Reply:
agreed, but this "extra care" makes sense, as now, the naming schemes for the result files are more indicative...
and overall, less data is saved, but more information...